Analysis/R/prepare_initial_values.R

```diff
@@ -317,11 +317,25 @@ if (config$obs_model == 3) {
         )
       }
       
-      # Small values of spatial random effects
-      init <- append(
-        init,
-        list(w = rnorm(stan_data$smooth_grid_N, 0, .1))
-      )
+      if (config$spatial_effect) {
```

Comment:
Should this be !config$spatial_effect?

Reply:
The warning message was in the wrong place. This fixed now.